Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:19-bullseye
```

Comment:
Replace with image for the `LTS` version of node.

Reply:
Done